test/ruby/test_array.rb

```diff
@@ -2197,6 +2197,16 @@ def test_repeated_permutation
     assert_empty(a.reject {|x| !x.include?(0)})
   end
 
+  def test_pluck
+    a = @cls[ {name: "Lewis"}, {name: "Steffan"} ]
+
+    assert_equal(a.pluck(:name), ["Lewis", "Steffan"])
+
+    b = @cls[ "string", {name: "Steffan"} ]
+
+    assert_raise(TypeError) { b.pluck(:name) }
+  end
```

Comment:
What does it do when a key is missing?

Reply:
It calls `hash[key]` so returns nil if the key is missing.